package.json

```diff
@@ -81,7 +81,7 @@
     "eslint-plugin-flowtype": "^2.35.0",
     "eslint-plugin-node": "^5.1.1",
     "file-url": "^2.0.2",
-    "jest": "^19.0.2",
+    "jest": "^21.1.0",
```

Comment:
oh, yeah, please don't change this? It's being held back for a reason (I haven't had time to figure out why jest 20+ breaks)